lib/Caxy/HtmlDiff/HtmlDiff.php

```diff
@@ -114,12 +133,17 @@ private function processInsertOperation($operation, $cssClass)
     {
         $text = array();
         foreach ($this->newWords as $pos => $s) {
+            $matchFound = false;
             if ($pos >= $operation->startInNew && $pos < $operation->endInNew) {
-                if ($s === '[[REPLACE_TABLE]]' && isset($this->newTables[$pos])) {
-                    foreach ($this->newTables[$pos] as $word) {
-                        $text[] = $word;
+                foreach ($this->specialElements as $specialElement) {
```

Comment:
Instead of looping through the specialElements array, you can just use `in_array`. This will eliminate the need for the $matchFound variable.

For example:

```php
if (in_array($s, $this->specialElements) && isset($this->newSpecialScript[$pos])) {
    foreach ($this->newSpecialScript[$pos] as $word) {
        $text[] = $word;
    }
} else {
    $text[] = $s;
}
```